deployment/test/flyte_generated.yaml

```diff
@@ -988,9 +969,8 @@ spec:
         - /etc/flyte/config/*.yaml
         - migrate
         - seed-projects
-        - flytesnacks
         - flytetester
```

Comment:
can we remove this one as well?

Reply:
It's used by the current version of flytetester image used in endtoend tests.... once that migrates off, then we can